### PR TITLE
Build warnings

### PR DESCRIPTION
**Pull request by ctrlaltdavid**
Fix some Windows build warnings

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 8803459a5b1a10907d39143c8e837d25ef52483d available at: https://gist.github.com/48508f916d931271b350<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: eeaa58c0e4bab9499808dceb48db11cb6c8ef22b available at: https://gist.github.com/48508f916d931271b350<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.dmg

File: libraries/model/src/model/Stage.cpp [hunk 1]
```diff
@@ -96,41 +96,41 @@ double moduloRange(double val, double minVal, double maxVal) {
     return rval * range + minVal;
 }
 
-const float MAX_LONGITUDE = 180.0f;
-const float MAX_LATITUDE = 90.0f;
+const double MAX_LONGITUDE = 180.0f;
```
Comment (ZappoMan) [inline libraries/model/src/model/Stage.cpp]: these should be doubles not floats... fix these warnings by keeping things in the float space.

Comment (ctrlaltdavid) [inline libraries/model/src/model/Stage.cpp]: Interesting that Visual Studio doesn't complain about defining const doubles with float values.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 361de790a85ffb8e7850923b8b5a3c30e2e97f93 available at: https://gist.github.com/48508f916d931271b350<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4807.dmg

**Comment by ctrlaltdavid** [comment]
@ZappoMan OK, I'll have another go in a new PR: for the float/double mismatch build warnings, instead of converting the floats to doubles, I'll convert the doubles to floats.

**Comment by ZappoMan** [comment]
thanks @ctrlaltdavid

**Pull request closed without merging**